courses/free-courses-si.md

```diff
@@ -36,11 +37,17 @@
 * [JavaScript Tutorial in Sinhala](https://youtube.com/playlist?list=PLYmyc7wRFoQjxkHAzHh1UIdU7ZdjTQvQt) - BestJobsLK
 
 
+### PHP
+
+* [PHP FULL COURSE IN SINHALA | 2022](https://www.youtube.com/watch?v=RdxtOQUflrk) - AUK Learning Center
+
+
 #### React
 
 * [Fundamentals \| React JS in Sinhala](https://youtube.com/playlist?list=PLvvtf05eMZ2DpDyWwmAjEuicvVxx4vIYB) - Fiqri Ismail
 * [MERN Stack Developer - Beginners](https://www.youtube.com/playlist?list=PLvfC6i-hEZBnqqF7giszuYI0iqenU5NY0) -  TechWithGeorge
 * [REACT | MERN CRUD App in Sinhala](https://youtube.com/playlist?list=PLtoqJbwHBeHzAooLCGOzYVE9mkAeCnT9y) - Haritha Weerathunga
+* [REACT JS FULL COURSE IN SINHALA | 2023](https://www.youtube.com/watch?v=tM02uzhHDPI&t=759s) -  AUK Learning Center
```

Comment:
please no all caps titles
Please read docs/CONTRIBUTING.md guidelines!

Reply:
Hi Hellman,

Thank you for your feedback on my pull request. I have made the requested changes to the code and believe that it now meets the project's standards.

Specifically, I updated the title of the pull request to be in line with the project's guidelines, and I read the CONTRIBUTING.md file and made sure to follow the guidelines there.

Please review the changes again at your earliest convenience. If there is anything else I can do to improve this pull request, please let me know.

Thank you,
Mahesh Abeykoon